content/overview/workflow/process.md

```diff
@@ -0,0 +1,209 @@
+
+### sharpness and local contrast
+
+A number of modules can be used to adjust the local contrast and sharpness of your image. Most of these modules aim to enhance the apparent contrast of edges and do not add "real" sharpness (they are not the same as lens deconvolution). You should take care when using these modules as most of them can introduce artifacts (such as halos) when settings are pushed too far:
+
+- the [_contrast equalizer_](../../module-reference/processing-modules/contrast-equalizer.md) module allows you to adjust contrast, limiting the effect to certain feature sizes. For example you can use it to increase the contrast of fine details without impacting larger-scale objects, or vice versa
+- the [_diffuse or sharpen_](../../module-reference/processing-modules/diffuse.md) module offers a number of presets for sharpening, lens deblurring and the addition of local contrast
+- the [_local contrast_](../../module-reference/processing-modules/local-contrast.md) module provides a simpler interface for quickly adding local contrast to your images -- just enabling the module or selecting one of the presets is often all that is required
+- the [_sharpen_](../../module-reference/processing-modules/sharpen.md) module is intended to re-introduce sharpness that was removed by your camera's anti-alias filter (if present) and can be enabled by default in [preferences > processing](../../preferences-settings/processing.md). The methods listed above are usually preferred to this legacy module.
+
+As with the modules mentioned in the previous section, you should take care when adding contrast to small-scale objects -- an image viewed at 100% is not a realistic representation of your final edit and local contrast adjustments are usually better judged when zoomed out.
+
+### reconstruct blown highlights
+
+While a well-exposed image will make post-processing much easier, darktable provides some tools handle blown highlights, in some cases even reconstructing lost colors or structure.
+
+The [_highlight reconstruction_](../../module-reference/processing-modules/highlight-reconstruction.md) module provides a very basic attempt to manage blown highlights. By default, if any color channel is clipped, this module will clip the other channels to white -- this prevents darktable from showing unnatural colors in the highlights. However it can be altered to attempt to reconstruct the lightness or color using any un-clipped channels. While this can re-introduce some detail, beware of the unrealistic colors that can result.
```

Comment:
I would mention here specifcally about the LCh option in hightlight reconstruction module. This can be a first step, falling back to filmic highlight recontruction if highlights.c doesn't give a satisfactory result.

Reply:
I think filmic prefers full data so is better if the highlight reconstruction module is disabled. I'm open to alternative wording if you want to suggest something.